src/test/java/hudson/plugins/copyartifact/CopyArtifactTest.java

```diff
@@ -333,7 +333,7 @@ public void testCopyAllWithFingerprints() throws Exception {
     public void testCopyAllWithoutFingerprints() throws Exception {
         FreeStyleProject other = createArtifactProject(),
                          p = createProject(other.getName(), null, "", "", false, false, false, false);
-        FreeStyleBuild s = rule.assertBuildStatusSuccess(other.scheduleBuild2(0, new Cause.UserIdCause()).get());
+        FreeStyleBuild s = rule.buildAndAssertSuccess(other);
         FreeStyleBuild b = p.scheduleBuild2(0, new Cause.UserIdCause()).get();
         rule.assertBuildStatusSuccess(b);
```

Comment:
Leaving more complex multiline idioms for proper tooling: https://github.com/openrewrite/rewrite-jenkins/issues/43